### DISCUSSION TITLE
Access to Global Campus as a Teacher

### DISCUSSION THREAD

**Discussion by prcolaco** [community/community#49516] [GitHub Education]
I submitted my application and got rejected due to the documentation. I sent a photo my ITIC (International Teacher ID Card) taken from the ITIC website, as the card is electronic only. Are ITIC/ISIC cards accepted as proof of teacher/student status?

I received the reply today, so I tried to apply again with another more quality photo of the card from the ITIC mobile app, but now it doesn't allow me to apply again. Is there any time I have to wait before applying again?

Thanks for your help.

**Comment by samus-aran** [comment]
Hi @prcolaco 

We're happy you're here. 👋 

Here are some tips to assist faculty applying to GitHub Global Campus:
1. If your school isn't a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution then we would recommend informal schools to apply first to the [GitHub Campus program](https://education.github.com/schools) to be verified.
2. We accept electronic IDs, but ensure you are able to demonstrate that your GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

Hope this helps.
